sdk/eventhub/azure-eventhubs/tests/asynctests/test_send_async.py

```diff
@@ -242,3 +243,6 @@ def batched():
         received.extend(r.receive(timeout=3))
 
     assert len(received) == 20
+
+    for r in receivers:
+        r.close()
```

Comment:
This should be awaited?

Reply:
receivers are from conftest. They're not async. The sender is async in this test method